src/Utils/Numbers.php

```diff
@@ -0,0 +1,98 @@
+<?php
+
+/**
+ * This file is part of the Nette Framework (https://nette.org)
+ * Copyright (c) 2004 David Grudl (https://davidgrudl.com)
+ */
+
+declare(strict_types=1);
+
+namespace Nette\Utils;
+
+use Nette;
+
+
+/**
+ * Floats tools library.
+ */
+class Numbers
+{
+	use Nette\StaticClass;
+
+	/** @var float */
+	private static $epsilon = 1e-9;
+
+
+	public static function isZero(float $value): bool
+	{
+		return abs($value) < self::$epsilon;
+	}
+
+
+	public static function isInteger(float $value): bool
+	{
+		return abs(floor($value) - $value) < self::$epsilon;
```

Comment:
imho `round` must be used instead of `floor`

Reply:
agree